projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -58,6 +58,83 @@ describe('Auth Guard', () => {
 });
 </code-example>
 
+#### Using Mock Selectors
+
+`MockStore` also provides the ability to mock individual selectors to return a passed value using the `overrideSelector()` method. When the selector is invoked by the `select` method, the returned value is overridden by the passed value, regardless of any state snapshot provided in `provideMockStore()`. 
+
+`overrideSelector()` returns a `MemoizedSelector`. To update the mock selector to return a different value, use the `MemoizedSelector`'s `setResult()` method.
+
+`overrideSelector()` supports mocking the `select` method (used in RxJS pipe) and the `Store` `select` instance method using a string or selector.
+
+Usage:
+
+<code-example header="auth.guard.ts">
```

Comment:
Instead of inlining the code, provide a `path` attribute.

```suggestion
<code-example header="auth-guard.service.ts" path="testing-store/src/app/auth-guard.service.ts">
```